sopel/module.py

```diff
@@ -261,6 +261,30 @@ def add_attribute(function):
     return add_attribute
 
 
+def action_commands(*command_list):
+    """Decorate a function to trigger on lines starting with "/me command".
+    This decorator can be used multiple times to add multiple rules. The
+    resulting match object will have the command as the first group, rest of
+    the line, excluding leading whitespace, as the second group. Parameters 1
+    through 4, separated by whitespace, will be groups 3-6.
+    Args:
+        command: A string, which can be a regular expression.
+    Returns:
+        A function with a new regular expression appended to the rule
+        attribute. If there is no rule attribute, it is added.
+    Example:
+        @action_commands("hello!"):
+            Would trigger on "/me hello!"
```

Comment:
I think it would be clearer to give a "proper" action as example, e.g. `/me says hello`.

The explanation could then say it would trigger "when a user 'says hello' as an action", or something. We should avoid doing `"/me hello!"` or anything else that might imply "/me" is part of the message. (This is also why I suggested an amendment to the one-line description.)

Reply:
Meh. I'll leave this to be addressed when this file gets its turn for #1565.